src/main/java/seedu/address/model/person/DateOfBirth.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.model.person;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a Person's date of admission in the address book.
+ * Guarantees: immutable;
+ */
+public class DateOfBirth {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Date of birth should be in the format of DD/MM/YYYY, and it should not be blank.";
+
+    public static final String VALIDATION_REGEX = "^(0[1-9]|[12][0-9]|3[01])/(0[1-9]|1[0-2])/\\d{4}$";
+
+    public final LocalDate dateOfBirth;
+
+    /**
+     * Constructs a {@code DateOfBirth}.
+     *
+     * @param dateOfBirth A valid date of birth.
+     */
+    public DateOfBirth(String dateOfBirth) {
```

Comment:
LocalDate.parse() by default follows the format YYYY-MM-DD. To define a custom DD/MM/YYYY format we can use DateTimeFormatter.ofPattern()

Reply:
or maybe we just specify a YYYY-MM-DD input